pages/docs/basic-features/layouts.mdx

```diff
@@ -18,8 +20,8 @@ The React model allows us to deconstruct a [page](/docs/basic-features/pages.md)
 ```jsx
 // components/layout.js
 
-import Navbar from './navbar'
-import Footer from './footer'
+import Navbar from "./navbar";
```

Comment:
We should keep the coding style from the origin docs, please consider disable the auto formatter during translation!

Reply:
Fixed